### PR TITLE
added imageSrcset.js

### PR DESCRIPTION
**Pull request by asad-rafter**

### CODE REVIEW

**Comment by asad-rafter** [comment]
@yuval-cloudinary, apologies, this file was missing in the previous PR. I just added please review

File: Cloudinary-pwa-kit/packages/pwa-cloudinary/overrides/app/components/footer/index.jsx
Comment (asad-rafter) [inline Cloudinary-pwa-kit/packages/pwa-cloudinary/overrides/app/components/footer/index.jsx:1]: @yuval-cloudinary, we were having issue when we directly access "cloudinary" object in Helmet tag, that's the reason we had to move it to outside of Helmet tag, it works fine this way

Comment (yuval-cloudinary) [reply Cloudinary-pwa-kit/packages/pwa-cloudinary/overrides/app/components/footer/index.jsx:1]: cool

**Review by yuval-cloudinary — APPROVED** [review]

File: Cloudinary-pwa-kit/packages/pwa-cloudinary/app/utils/imageSrcset.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { cloudinary } from '../../config/default'
+
+/**
+ * Upate the tracking params in srcsets
+ *
+ * @param {string} srcset - srcset.
+ * @returns {string} - The ImageGroup matching the search criteria
+ */
+export const updateTrackingParam = (srcset) => {
+    let splits = srcset.split(' ')
```
Comment (yuval-cloudinary) [inline Cloudinary-pwa-kit/packages/pwa-cloudinary/app/utils/imageSrcset.js]: please use const

File: Cloudinary-pwa-kit/packages/pwa-cloudinary/app/utils/imageSrcset.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    let newSrcset = splits.map((split) => {
```
Comment (yuval-cloudinary) [inline Cloudinary-pwa-kit/packages/pwa-cloudinary/app/utils/imageSrcset.js]: please use const

**Review by yuval-cloudinary — CHANGES_REQUESTED** [review]

**Review by yuval-cloudinary — APPROVED** [review]

**Review by yuval-cloudinary — COMMENTED** [review]
Approved

**Pull request merged**